R/declare_test.R

```diff
@@ -78,13 +80,64 @@
 #' )
 #' }
 #' 
-declare_test <- declare_estimator
+#' @seealso See \code{\link{declare_estimator}} for documentation of the \code{model_handler} function.
+#' 
+declare_test <- 
+  make_declarations(
+    tidy_test(model_handler),
+    step_type = "estimator",
+    causal_type = "estimator",
```

Comment:
Depending how far you want to take this, you could make a new causal_type also, so that "tests" are aggregated and tagged differently from estimates. That would allow different sets diagnostics on each (eg no bias/coverage on tests).  Someone will for sure ask how to not include a bunch of NA columns for test-only designs.